src/components/SponsorTimeEditComponent.tsx

```diff
@@ -240,6 +282,18 @@ class SponsorTimeEditComponent extends React.Component<SponsorTimeEditProps, Spo
         );
     }
 
+    wheelUpOrDown(e: React.WheelEvent): boolean{
+        // Returns true if the wheel scrolls up and false if the wheel scrolls down
+        console.log("wheelUpOrDown")
+        if (e.deltaY < 0)
+        {
+            return true;
+        }
+        else if (e.deltaY > 0)
+        {
+            return false;
+        }
```

Comment:
```suggestion
        return e.deltaY < 0;
```

Reply:
I'm not sure if `deltaY` can be = 0

Reply:
[It seems like it can be 0](https://www.w3schools.com/jsref/event_wheel_deltay.asp)

> otherwise 0

I dont know how to handle this. Could the whole thing crash if it is 0?